docs/support/tor.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: default
+title: Tor
+parent: Supporting Applications
+nav_order: 720
+---
+## Tor: Private Networking
+
+Running the BitBox Base means connecting to other Bitcoin and Lightning Network peers.
+If done over the regular internet protocol (IPv4), your IP address is visible to all your communication partners.
+As IP addresses can be located geographically, this means revealing your physical location, from the general area down to individual city blocks (the granularity depends on how your internet service provider manages IP addresses).
+You can easily locate yourself using services like <https://iplocation.io> or <http://www.infosniper.net>.
+
+Revealing your location with the information that you are using Bitcoin is not a good idea. This is why the BitBox Base uses [The Onion Router](https://www.torproject.org/) (Tor) to communicate to other Bitcoin peers privately and without revealing your real IP address.
+
+By creating "hidden services", Tor can also be used to securely access your BitBox Base from outside your local network without having to configure any networking devices, such as port-forwarding on your router. 
+
+Please not that only Bitcoin-related communication is routed over Tor.
+
+### Installation
+
+Tor is installed using the official repository at `https://deb.torproject.org/torproject.org`, over a TLS connection secured by the official Tor signing key.
```

Comment:
I think it's not quite accurate to say that the `official Tor signing key` is what secures the `TLS connection` (transport-layer security is separate from code signing).. maybe:

> Tor is installed using the official repository at `https://deb.torproject.org/torproject.org`, and the official PGP signing key for the Tor project is used to verify the integrity of the package.

Reply:
Changed to 
> Tor is installed using signed Debian packages from the official repository at `https://deb.torproject.org/torproject.org`.

Reply:
Seems to have been done in https://github.com/digitalbitbox/bitbox-base/pull/66. Resolving this thread.